config/datocms/migrations/1734806654_previewLinks.ts

```diff
@@ -0,0 +1,64 @@
+import { Client } from '@datocms/cli/lib/cma-client-node';
+
+export default async function (client: Client) {
+  console.log('Manage upload filters');
+
+  console.log('Install plugin "Model Deployment Links"');
+  await client.plugins.create({
+    id: 'MKba9NT5QBKZaeI4HcERwA',
+    package_name: 'datocms-plugin-model-deployment-links',
+  });
+  await client.plugins.update('MKba9NT5QBKZaeI4HcERwA', {
+    parameters: { datoApiToken: process.env.DATOCMS_PREVIEW_API_TOKEN },
```

Comment:
I'm struggling where to define this token. Similar issue as with the #170. Maybe rather add it to the getting started documentation that these tokens need to be created and configured in the CMS?
(this is why currently all the CI checks are failing).

Reply:
Decided to handle this in the getting started documentation instead.